data/transition-sites/scotlandoffice.yml

```diff
@@ -1,9 +1,9 @@
 ---
 site: scotlandoffice
-whitehall_slug: scotland-office
+whitehall_slug: office-of-the-secretary-of-state-for-scotland
 host: www.scotlandoffice.gov.uk
 tna_timestamp: 20130130150421
 homepage_furl: www.gov.uk/scotland-office
-homepage: https://www.gov.uk/government/organisations/scotland-office
+homepage: https://www.gov.uk/government/organisations/office-of-the-secretary-of-state-for-scotland
 options: --query-string title:attachment
-css: scotland-office
+css: office-of-the-secretary-of-state-for-scotland
```

Comment:
I think this will break the CSS for logos on the archive pages as the slug changed - you'll have to update https://github.com/alphagov/bouncer/blob/master/public/gone.css#L359-L372 to be the new slug too. And the same for DDCMS, in https://github.com/alphagov/bouncer/blob/master/public/gone.css#L247.

Reply:
Raised a PR here: https://github.com/alphagov/bouncer/pull/147